### PR TITLE
1_FireFox

### PR DESCRIPTION
**Pull request by FireMedusa**
- Completed again.

### CODE REVIEW

**Comment by oddcommits** [comment]
@FireMedusa resolve conflicts

File: README.md [hunk 1]
```diff
@@ -88,4 +88,5 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 
 - [John Doe](./john-doe/README.md)
 - [Sebastian Harvey](./jsstar/README.md)
+- [Yusuke Nishikado](./yusukenishikado/README.md)
```
Comment (oddcommits) [inline README.md:91]: `whitecollin` was added earlier than you

File: yusukenishikado/README.md
Comment (oddcommits) [inline yusukenishikado/README.md:1]: good README!

File: yusukenishikado/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Laravel
+- Mern Stack
+
+## Email Address
+
+My email address is [yusukenishikado96@gmail.com]
+
+## Applications
+
+- [todo](./todo/README.md)
```
Comment (oddcommits) [inline yusukenishikado/README.md:21]: don't point README.md, but the whole todo directory

**Review by oddcommits — CHANGES_REQUESTED** [review]
Good job, minor fixes

File: yusukenishikado/README.md (hunk 2 above)
Comment (FireMedusa) [reply yusukenishikado/README.md]: I resolved problem

File: README.md [hunk 3]
```diff
@@ -89,5 +89,6 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 - [John Doe](./john-doe/README.md)
 - [Sebastian Harvey](./jsstar/README.md)
 - [White Collin](./whitecollin/)
+- [Yusuke Nishikado](./yusukenishikado/README.md)
```
Comment (oddcommits) [inline README.md:92]: your name should be placed after Lucas

File: README.md [hunk 4]
```diff
@@ -89,5 +89,6 @@ This repository is designed to help newbies practice using Git and GitHub by cre
...
 - [Donald Ross](./donald-ross/)
-- [Lucas Marvel](./lucas/README.md)
+- [Lucas Marvel](./lucas/README.md)
```
Comment (oddcommits) [inline README.md:94]: don't remove a new line at the end

**Review by oddcommits — CHANGES_REQUESTED** [review]

**Review by oddcommits — APPROVED** [review]
it looks solid!

**Pull request merged**